src/nl/rubensten/texifyidea/intentions/latexmathtoggle/MathEnvironmentEditor.kt

```diff
@@ -0,0 +1,119 @@
+package nl.rubensten.texifyidea.intentions.latexmathtoggle
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import nl.rubensten.texifyidea.util.*
+
+class MathEnvironmentEditor(private val oldEnvName: String,
+                            private val newEnvName: String,
+                            val editor: Editor,
+                            val environment: PsiElement) {
```

Comment:
Reformat to Kotlin code standards:
```
class TooManyArguments(
    var first: Byte,
    val second: List<Set<Map<Short, Int>>> = emptyList(),
    private var third: Long
) {
    // Code...
}
```
------------------------
`Env` -> `Environment`